internal/daysteps/daysteps.go

```diff
@@ -13,8 +19,45 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("Недопустимый формат даннных")
+	}
+	if strings.Contains(parts[0], " ") {
+		return 0, 0, fmt.Errorf("пробелы в начале/вконце")
+	}
+	steps, err := strconv.Atoi(strings.TrimSpace(parts[0]))
+	if err != nil {
+		return 0, 0, fmt.Errorf("Недопустимый формат шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```